docs/azmcp-commands.md

```diff
@@ -223,6 +223,12 @@ azmcp applens resource diagnose --subscription <subscription> \
                                 --resource <resource>
 ```
 
+### Azure Cli Generate Operations
+```bash
+# Generate an Azure Cli command based on user intent
+azmcp extension cli generate --cli-type <cli-type> --intent <intent>
```

Comment:
should be azcli as there are many clis

Reply:
There is another service that can generate Azure PowerShell commands. It will be another valid value for cli-type argument.